esx_service/vmdk-opsd

```diff
@@ -112,32 +113,65 @@ stop() {
    if [ -n "${PID}" ]; then
       echo "Stopping $OPSD_TAG with PID=${PID}"
       kill -$SIGTERM ${PID}
+
       local COUNT=0
       while kill -0 ${PID} > /dev/null 2>&1; do
          sleep ${WAIT_KILL_SEC};
+         echo "Waiting for operations in flight to complete, please do not interrupt..."
          if [ $((COUNT++)) -gt ${NUM_RETRY} ]; then
-            echo "Sending kill -${SIGKILL} to ${PID} after kill -${SIGTERM} did not stop ${PID}"
-            kill -${SIGKILL} ${PID}
+            echo "Kill -${SIGTERM} did not stop ${PID}"
             break
```

Comment:
Why remove the kill, rather than have the user issue multiple commands just to kill a process, better do it here. When asked to stop the service is allowed to drain commands and when that doesn't complete the process is killed - which should be rare.

Basically, the user wants to stop the process - the user  isn't going to debug the process anyway. So stop should do the job. User will most likely do a ps -c and grep the PID and kill it if one command doesn't handle.

Reply:
I don't have strong opinion about this - either way seems acceptable to me. @msterin do you have any opinion about this?

Reply:
If stopping fails because drain failq, the whole command fails will clear message. User can always do force kill.if needed, but we should not force kill after a request for soft stop, IMO

Reply:
Yes. For example, in a long running task (e.g. formatting a huge disk), we may not want to force kill the operation in a stop request. It's better to let users do it explicitly by themselves.